cmake/features.cmake

```diff
@@ -54,3 +54,7 @@ ie_dependent_option (ENABLE_FASTER_BUILD "Enable build features (PCH, UNITY) to
 # Type of build, we add this as an explicit option to default it to ON
 # FIXME: Ah this moment setting this to OFF will only build ngraph a static library
 ie_option (BUILD_SHARED_LIBS "Build as a shared library" ON)
+
+ie_option_enum(SELECTIVE_BUILD "Enable OpenVINO conditional compilation or statistics collection" OFF
```

Comment:
nit: Can we add that `SELECTIVE_BUILD=ON` requires the specialization of `SELECTIVE_BUILD_STAT` option?
And also add information about what is `SELECTIVE BUILD_STAT`.

Reply:
Inside openvino/conditional_compilation/CMakeLists.txt:31 I added error message generation for SELECTIVE BUILD_STAT. Is it enough?

Reply:
Hmmm, it looks like an cmake option. And usual we have a description of all CMake options in this file.

Reply:
Description added.